src/NosCore.Parser/Parsers/ItemParser.cs

```diff
@@ -920,11 +926,11 @@ public void Parse(string folder)
 
                                         break;
 
-                                    case 204:
-                                        item.EffectValue = 10000;
+                                    case ItemEffectType.SpecialistMedal:
+                                        item.EffectValue = 30_000;
```

Comment:
why another value ?

Reply:
Based in UK description it gives you 30.000 Specialist Points every day, no 10.000